Makefile

```diff
@@ -1,38 +1,20 @@
-PREFIX=/usr/local
-BINDIR=${PREFIX}/bin
-DESTDIR=
-BLDDIR = build
-BLDFLAGS=
-EXT=
-ifeq (${GOOS},windows)
-    EXT=.exe
-endif
+GOVERSION := go1.19.1
```

Comment:
Please do not use go version directly

Reply:
It's best place to specify go version, since otherwise it should be specified in more than one place.

Reply:
Where the places?

Reply:
I guess it will be debian/rules and cloudlinux-nsqd.spec